src/commands/deploy.ts

```diff
@@ -54,41 +60,116 @@ $ bump deploy FILE --dry-run --doc <doc_slug> --token <your_doc_token>
   async run(): Promise<void> {
     const { args, flags } = this.parse(Deploy);
 
-    const [dryRun, documentation, token, hub, autoCreate, documentationName, branch] = [
+    const [
+      dryRun,
+      documentation,
+      token,
+      hub,
+      autoCreate,
+      interactive,
+      documentationName,
+      branch,
+    ] = [
       flags['dry-run'],
-      /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
-      flags.doc!,
+      flags.doc,
       /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
       flags.token!,
       flags.hub,
       flags['auto-create'],
+      flags['interactive'],
       flags['doc-name'],
       flags.branch,
     ];
     const file = await statSync(args.FILE);
 
-    if (file.isDirectory()) {
-      throw new Error(
-        'Deploy with a directory is not yet implemented. Please wait while we are working on it!',
-      );
-    } else {
-      await this.deploySingleFile(
+    if (file.isDirectory() && hub) {
```

Comment:
You've already seen it 😏, but the case (directory && !hub) should be managed separately.

Reply:
:grin: yeh I'll tackle that now!